AirCasting/Models/SessionExtension.swift

```diff
@@ -19,7 +19,9 @@ extension SessionEntity {
     
     var pm1Stream: MeasurementStreamEntity? {
         let match = measurementStreams?.first(where: { (stream) -> Bool in
-            (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam3-PM1"
+            (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam3-PM1" ||
+                (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam2-PM1" ||
+                    (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam1-PM1"
```

Comment:
Why do we need those AirBeams if we can't connect with them?

Reply:
Because then PM1 stream from AirBeam2 or 1 is not recognised and stream goes the way, where PM1 is last and it should't be

Reply:
Well, that's not what I meant but nvm, now I get it - we might have sessions recorded with AB1 and AB2 on Android.